tests/test.py

```diff
@@ -4553,6 +4554,7 @@ def get_context_data(self):
         context = {
             'id': get_random_id(),
             'default_client_id': get_pregenerated_random_data().encode('ascii'),
+            'number': 123456,
```

Comment:
I'd suggest to use `get_random_id` or something to get a random integer here, just to avoid breaking the pattern of using random inputs, not fixed one. However, I don't insist.

Reply:
Done. Had to add another line to preserve the value between calls to `get_context_data()`